apps/linode-marketplace-openlitespeed-django/roles/post/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+-
+# motd and creds gen
+- name: copy MOTD template to /etc/motd
+  template:
+    src: templates/motd.j2
+    dest: /etc/motd
+
+- name: writing creds into file
+  copy:
+    dest: /home/{{ username }}/.credentials
+    mode: '0600'
+    content: |
+      Web Admin password: {{ litespeed_password }}
```

Comment:
Can you verify that this is not a base64 digest. I believe that we might need to decode the string content of the var. Maybe clean up and remove `/root/.litespeed_password`.

Reply:
There's nothing here that indicates it's base64:
https://docs.litespeedtech.com/cloud/images/django/#how-to-access-the-installed-software

When logging in through http://<ip_address>:7080 to the openlitespeed dashboard, it works with the creds `admin` and the password at /root/litespeed_password